modules/mobile/app/services/mobile/v0/appointments/appointment_requests_service.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    module Appointments
+      # connects to appointment request service
+      class AppointmentRequestsService < VAOS::SessionService
+        # fetches the past 90 days of appointment requests
```

Comment:
We always fetch 90 days. Due to the way we handle caching in the controller, we always request records from at least a year ago through at least a year from now, so the past 90 days are always required.

Reply:
Do we need our own service because of the date range requirements on mobile? We have our own service for the appointments list because we parallelize it. For canceling appointments, we re-use their service. It's cool if we really do need our own service, but we should lean towards reusing services as much as possible.

Reply:
I'll look into that. I'd also prefer to re-use code.

Reply:
I've done some playing around with it, and I believe I can use their service. It's a little funky because their service handles errors differently, but I think it should work. Thanks for the suggestion.